client.go

```diff
@@ -32,13 +32,16 @@ type Client interface {
 	// GetActive returns all of the active key versions for the knox key.
 	// This should be used for receiving relationships like verifying or decrypting.
 	GetActive() []string
+	//Get knox Key Object. The Key Object returned can be further used to query versionIDs, ACLs, and other attributes
```

Comment:
Let's follow Go's documentation convention:

```suggestion
	// GetKeyObject returns the full key object, including versions, ACLs, and other attributes.
```

Reply:
Thanks @jparise . Updated the documentation.